uniro_frontend/src/pages/demo.tsx

```
@@ -6,18 +6,57 @@ import Map from "../component/Map";
 import DestinationIcon from "../assets/map/destination.svg?react";
-import { useState } from "react";
+import { useEffect, useState } from "react";
 import ReportButton from "../components/map/reportButton";
 import { CautionToggleButton, DangerToggleButton } from "../components/map/floatingButtons";
+import { useMutation, useQuery, useSuspenseQuery } from "@tanstack/react-query";
+import { getFetch, postFetch, putFetch } from "../utils/fetch/fetch";
+import SuspenseMapComponent from "../component/SuspenseMap";
+import { getMockTest } from "../utils/fetch/mockFetch";
+import SearchNull from "../components/error/SearchNull";
+import Offline from "../components/error/Offline";
+import Error from "../components/error/Error";
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)